File: 5. Arrays/2. Java/findingelement.java
import java.util.Scanner;
public class findingelement {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);        
        int [] arr = new int [5];
        System.out.println("Enter the integer to be found");
        int element = sc.nextInt();
        System.out.println("Enter the elements of the array: ");
        for(int i=0; i<arr.length;i++){
            arr[i] = sc.nextInt();
        }
        for(int i=0; i<arr.length;i++){
            if(arr[i] == element){
                System.out.println("Found");
            }
            else{
                System.out.println("Not Found");
            }
        }
        sc.close();
    }
}

File: 5. Arrays/1. Cpp/findingelement.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
	int n;
	cout<<"Enter the number of elements: "<<endl;
	cin>>n;
	int arr[n];
	int element;
	cout<<"Enter the element to be found: "<<endl;
	cin>>element;
	cout<<"Enter the elements: "<<endl;
	for(int i=0;i<n;i++) cin>>arr[i];
	cout<<"Processing......"<<endl;
	for(int i=0;i<n;i++){
		if(arr[i] == element){
			cout<<"Found"<<endl;
		}
		else{
			cout<<"Not Found"<<endl;
		}
	}
	return 0;
}
